stl/inc/yvals_core.h

```diff
@@ -35,6 +35,7 @@
 // P0646R1 list/forward_list remove()/remove_if()/unique() Return size_type
 // P0653R2 to_address()
 // P0655R1 visit<R>()
+// P0660R10 <stop_token> and jthread

```

Comment:
Should capitalize `And` to be consistent with #32. (These cleaned-up titles currently follow the somewhat nonstandard convention of applying title case to everything except identifiers.)